mesos-docker/README.md

```diff
@@ -228,3 +228,16 @@ An example of using the file would be:
 ```
 
 Note: Using a configuration file overrides the number of slaves started and takes precedence over --number-of-slaves or the default value.
+
+### Setting docker mesos lib version
+
+By default docker images should have the latest mesos library as it is defined here:
+http://mesos.apache.org/downloads/.
+
+If there is an issue with some mesos release you can always fallback by setting the docker mesos lib version
+through either command line with the parameter * update-image-mesos-at-version* or through the environmental variable
```

Comment:
I think there's a space between `*` and `update-image-mesos-at-version`, which makes the formatting fail.